modules/Data_Visualization/lab/Data_Visualization_Lab_Key.Rmd

```diff
@@ -153,7 +153,7 @@ facet_plot <- ggplot(bike_agg_2, mapping = aes(
 facet_plot
 ```
 
-**Bonus / Extra practice**: Modify `facet_plot` to remove the legend (hint use `theme()`) and change the names of the axis titles to be "Number of bike lanes" for the y axis and "Date bike lane was installed" for the x axis. 
+**Bonus / Extra practice**: Modify `facet_plot` to remove the legend (hint use `theme()` and the `legend.position` argument) and change the names of the axis titles to be "Number of bike lanes" for the y axis and "Date bike lane was installed" for the x axis. 
```

Comment:
Added it in the key as well. Sometime I should help you automate the notebook solution -> no solution creation. I have answers removed automatically and for datatrail project notebook solutions. (but this can be done at a later time).

Reply:
would be great! although we add extra alternatives though...so that would be different

Reply:
Agree, this is definitely on the list :)